po_lines_location

```diff
@@ -0,0 +1,40 @@
+DROP TABLE IF EXISTS po_lines_locations;
+
+-- Create a local table for Purchase Order Line Locations including location quantity and type.
+
+CREATE TABLE po_lines_locations AS
+WITH pol_location AS (
+    SELECT
+        pol.id::uuid AS pol_id,
+        jsonb_extract_path_text(locations.data, 'locationId') AS pol_loc_id,
+        jsonb_extract_path_text(locations.data, 'quantity') AS pol_loc_qty,
+        jsonb_extract_path_text(locations.data, 'quantityElectronic') AS pol_loc_qty_elec,
+        jsonb_extract_path_text(locations.data, 'quantityPhysical') AS pol_loc_quant_phys
```

Comment:
Perhaps `pol_loc_quant_phys` could named as `pol_loc_qty_phys` just for naming consistency.